spec/features/vehicle_spec.rb

```diff
@@ -21,7 +21,6 @@
     fill_in 'Chassi', with: '9BWZZZ377VT004251'
```

Comment:
It is you on, but as it were in allowed it could be if you add one more controller test doing a "override" of this :P